app/types/marc_nokogiri_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class MarcNokogiriType < NokogiriType
+  def cast(value)
+    result = super
+    return result if result.blank? || result.root.blank? || result.root.namespaces.present?
+    result.root['xmlns'] = 'http://www.loc.gov/MARC21/slim'
```

Comment:
When do we wind up here?  When we parse a MARC XML file that doesn't have namespaces declared properly?